src/components/ButtonHelper/index.tsx

```diff
@@ -11,6 +11,10 @@ const UnStyledButton = styled.button`
   outline-color: ${({ theme }) => theme.colors.icon};
   display: flex;
   align-items: center;
+
+  span:hover {
+    filter: brightness(50%);
```

Comment:
Works well for the pencil icon but not so well for the trash bin:
<img width="153" alt="Screenshot 2021-06-14 at 13 28 04" src="https://user-images.githubusercontent.com/381895/121885370-60ae2e80-cd14-11eb-878e-2a2ea6661aa0.png">

Reply:
Yes, I'm working in other approach in SRC based on the original design:
https://app.zeplin.io/project/5b83b4fa75f90943c86ed14f/screen/5e7394b2dc79fdbde6d4d1f8
https://app.zeplin.io/project/5b83b4fa75f90943c86ed14f/screen/5f5b904b0faf6c76b20961f3
![image](https://user-images.githubusercontent.com/622217/121894084-18334880-ccf5-11eb-8dec-361037cfa051.png)
On hover, the icon color remains the same and a circle container is added in the background.